hatchery/efs.go

```diff
@@ -19,6 +19,9 @@ type EFS struct {
 
 func (creds *CREDS) getEFSFileSystem(userName string, svc *efs.EFS) (*efs.DescribeFileSystemsOutput, error) {
 	fsName := strings.ReplaceAll(os.Getenv("GEN3_ENDPOINT"), ".", "-") + userToResourceName(userName, "pod") + "fs"
+	if len(fsName) > 63 {
+		fsName = fsName[:63]
+	}
```

Comment:
i think this could cause bugs in the future if multiple truncated strings end up being the same? i'll suggest hashing to get a shorter string, but i know it's nice to see the full name for debugging - is that important? If it is, maybe just truncating is fine for now, but could we move this logic to a function so that if this starts causing issues, we can update it in a single place